shared/src/commonMain/resources/MR/base/colors.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- format: #RRGGBB[AA] or 0xRRGGBB[AA] or RRGGBB[AA] where [AA] - optional -->
+    <color name="backgroundColor">#15151A</color>
+    <color name="white">#FFFFFF</color>
```

Comment:
Do we really need this color declaration - `white`? can't we access the default white color per platform? 🤔

Reply:
Updated this name to `primary` for clarity.